x-pack/plugins/cases/public/components/all_cases/all_cases_list.test.tsx

```diff
@@ -836,4 +859,14 @@ describe('AllCasesListGeneric', () => {
 
     waitForComponentToUpdate();
   });
+
+  it('should hide the alerts column if the alert feature is disabled', async () => {
+    const { queryByText } = render(
+      <TestProviders features={{ alerts: false }}>
+        <AllCasesList />
+      </TestProviders>
+    );
+
+    expect(queryByText('Alerts')).toBe(null);
```

Comment:
please use a testId here.

Reply:
It is not recommended by the React Testing Library to use ids if not needed. See here: https://testing-library.com/docs/queries/about#priority

Reply:
you could use `queryByTestId` and assert that the query returned null.

Reply:
I know that. What I try to say is that I don't think we should test by ids if not needed. Check the link I posted.

Reply:
I've seen it before. But I see we ids everywhere in kibana. The reason I go against using text is because it is easy to put even a sample text that says "Alerts" in one of the mocks and this will start failing.

Reply:
Ok, I will use the ids.